srcpkgs/samsung-unified-driver/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'samsung-unified-driver'
 pkgname=samsung-unified-driver
-version=1.00.37
-revision=4
+version=1.00.39
+revision=1
 archs="i686 x86_64"
 create_wrksrc=yes
 depends="cups ghostscript libusb-compat sane"
 short_desc="Unified Linux Driver for Samsung printers and scanners"
 maintainer="Thomas Bernard <thomas@famillebernardgouriou.fr>"
 license="custom:Propietary"
-homepage="https://www.samsung.com"
-distfiles="https://downloadcenter.samsung.com/content/DR/201512/20151210091120064/uld_v${version}_00.99.tar.gz"
-checksum="6b85253ea0bb51d241f6fd665ff0d39464cdad87084802a77a385c707fa2c664"
+homepage="https://support.hp.com"
+distfiles="https://www.bchemnet.com/suldr/driver/UnifiedLinuxDriver-${version}.tar.gz"
```

Comment:
why did these urls change?

Reply:
Samsung sold his printer business unit to HP, apparently already in 2016 (https://news.samsung.com/global/samsung-electronics-announces-divestment-of-printing-business-unit-and-nomination-of-jay-y-lee-to-the-board-of-directors).
That's why the first url changed.

Now, one could set the download url to hp as well, but there is no universal download link to the drivers, one has to choose a model to download the drivers - which are the same for all models.

As the other url showed up as first result when searching for driver downloads (https://www.bchemnet.com/suldr/suld.html), I checked it out and decided to use their download url because if one would choose the download link for one specific printer model on the hp page and that model gets removed for whatever reason one would have to adjust the download url. That's why I decided to use their download url because it's universal, not tied to a specific model.

Btw., it was only after I had installed the drivers manually that I learned that the drivers are also in the void repo. Building and installing the drivers the xbps way has the same outcome, as one would expect.

Reply:
ok, https://www.bchemnet.com/suldr/suld.html would be a more useful homepage

Reply:
Ok, thanks.

I edited the commit and pushed the new commit, but it doesn't seem right, as it didn't replace the first one, which I hoped to be able to do.

Reply:
do `git reset --soft HEAD~1`, then `git commit --amend` and it should merge the commits

Reply:
I'm working with the github website...
I guess I close this pull request an make a new one.

What's about the lint error regarding the email address? Is that an issue that needs fixing and I can implement before I make the the new pull request?

Reply:
yes, you need to fix both. you don't need to make a new pull request, you just need to use git locally instead of doing everything on github.

Reply:
Still not sure how I can solve this lint issue: "authored by noreply email"
Not even sure what that means, as the email doesn't contain "noreply". Does that test check somehow if the email address is alive, and this one isn't?
Should the maintainer info then be changed to the usual orphaned info?

Reply:
commits created on github use a special email that's `yourusername@noreply.users.github.com`. we don't want this on commits.

Reply:
Thank, now I understand. It's not about the email address in the template file but of the commit.
Ok, I'll try to do the local/cli way.

Reply:
Finally figured out and managed to get the (hopefully) merged / amended commit from local to remote...

Hope, this time all is fine and correct.

Reply:
Thanks again for your help, Abby!